templates/python-empty/README.md

```diff
@@ -0,0 +1,15 @@
+# Empty Python Actor template
+
+The Empty Python Actor template is a starting point for creating a new Python actor on the Apify platform. It provides a basic structure for the actor and allows you to easily add your own functionality. 
+
+## Getting Started
+
+Start a new web scraping project quickly and easily in Python with our empty project template. It provides a basic structure for the Actor with [Apify SDK](https://docs.apify.com/sdk/python/) and allows you to easily add your own functionality.
```

Comment:
Isn't this a duplicate of the first paragraph?

Reply:
So should I remove it?